lib/galaxy/config/schemas/config_schema.yml

```diff
@@ -1520,7 +1520,20 @@ mapping:
         default: interactivetools_map.sqlite
         path_resolves_to: data_dir
         desc: |
```

Comment:
Can you mention here that the value is ignored if the  `interactivetools_map_sqlalchemy` property is set? (I think we should include this in both property descriptions, not just one)

Reply:
Addressed in 018e5417